HighlightUtilities/_themeColors.php

```diff
@@ -0,0 +1,479 @@
+            1 => 255,
+            2 => 255,
+        ),
+        'magula' => array(
+            'r' => 244.0,
+            'g' => 244.0,
+            'b' => 244.0,
+        ),
+        'mono-blue' => array(
+            'r' => 234.0,
+            'g' => 238.0,
+            'b' => 243.0,
+        ),
+        'monokai-sublime' => array(
+            'r' => 35.0,
+            'g' => 36.0,
+            'b' => 31.0,
+        ),
+        'monokai' => array(
+            'r' => 39.0,
+            'g' => 40.0,
+            'b' => 34.0,
+        ),
+        'night-owl' => array(
+            'r' => 1.0,
+            'g' => 22.0,
+            'b' => 39.0,
+        ),
+        'nord' => array(
+            'r' => 46.0,
+            'g' => 52.0,
+            'b' => 64.0,
+        ),
+        'obsidian' => array(
+            'r' => 40.0,
+            'g' => 43.0,
+            'b' => 46.0,
+        ),
+        'ocean' => array(
+            'r' => 43.0,
+            'g' => 48.0,
+            'b' => 59.0,
+        ),
+        'paraiso-dark' => array(
+            'r' => 47.0,
+            'g' => 30.0,
+            'b' => 46.0,
+        ),
+        'paraiso-light' => array(
+            'r' => 231.0,
+            'g' => 233.0,
+            'b' => 219.0,
+        ),
+        'pojoaque' => array(
+            0 => 255,
+            1 => 255,
+            2 => 255,
+        ),
+        'purebasic' => array(
+            'r' => 255.0,
+            'g' => 255.0,
+            'b' => 223.0,
+        ),
+        'qtcreator_dark' => array(
+            'r' => 0.0,
+            'g' => 0.0,
+            'b' => 0.0,
+        ),
+        'qtcreator_light' => array(
+            'r' => 255.0,
+            'g' => 255.0,
+            'b' => 255.0,
+        ),
+        'railscasts' => array(
+            'r' => 35.0,
+            'g' => 35.0,
+            'b' => 35.0,
+        ),
+        'rainbow' => array(
+            'r' => 71.0,
+            'g' => 73.0,
+            'b' => 73.0,
+        ),
+        'routeros' => array(
+            'r' => 240.0,
+            'g' => 240.0,
+            'b' => 240.0,
+        ),
+        'school-book' => array(
+            0 => 255,
+            1 => 255,
+            2 => 255,
+        ),
+        'shades-of-purple' => array(
+            'r' => 45.0,
+            'g' => 43.0,
+            'b' => 87.0,
+        ),
+        'solarized-dark' => array(
+            'r' => 0.0,
+            'g' => 43.0,
+            'b' => 54.0,
+        ),
+        'solarized-light' => array(
+            'r' => 253.0,
+            'g' => 246.0,
+            'b' => 227.0,
+        ),
+        'sunburst' => array(
+            'r' => 0.0,
+            'g' => 0.0,
+            'b' => 0.0,
+        ),
+        'tomorrow-night-blue' => array(
+            'r' => 0.0,
+            'g' => 36.0,
+            'b' => 81.0,
+        ),
+        'tomorrow-night-bright' => array(
+            0 => 255,
+            1 => 255,
+            2 => 255,
```

Comment:
Apparently `#fff` is getting parser without the r g b keys. Perhaps this should get normalized for highlight.php?

Reply:
Ahh oops, it wasn't that it was being parsed incorrectly, I just gave it the wrong default values when no color was given. Technically it's transparent but I'm just assigning it white. Should we indicate transparent somehow?

Reply:
I suppose a default of white is correct, since that is the browser default. That is unless there are dark schemes that also have transparent backgrounds?

Reply:
I don't think any dark themes have a transparent background. I was only thinking about [`@media(prefers-color-scheme)`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) but I think at that point, it should be up to the theme designer to get colors right.